NineChronicles.Headless.AccessControlCenter/AccessControlService/MutableRedisAccessControlService.cs

```diff
@@ -16,29 +15,28 @@ public MutableRedisAccessControlService(string storageUri)
         {
         }
 
-        public void DenyAccess(Address address)
+        public void AddTxQuota(Address address, int quota)
```

Comment:
I think we need to set a quota limit, perhaps... at 10?

Reply:
@Atralupus Should I add the same limitation(https://github.com/planetarium/NineChronicles.Headless/pull/2278/commits/7d3a51adca1d9bf5383d26f6a534c9852a2948d0) here?

Reply:
yes, right